resources/snap_packs/snap_packs.yml

```diff
@@ -0,0 +1,353 @@
+post:
+  operationId: snap_pack_create
+
+  summary: Create
+
+  description: >-
+    Creates a new snap_pack given information
+
+  tags:
+    - Snap Packs
+
+  parameters:
+    - $ref: "../../shared/parameters/idempotency.yml#/idem-header"
+    - $ref: "../../shared/parameters/idempotency.yml#/idem-query"
+    - $ref: "parameters/lob_version.yml#/lob-version"
+
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: "models/snap_pack_editable.yml"
+        example:
+          description: Demo Snap Pack Job
+          to: adr_bae820679f3f536b
+          from: adr_210a8d4b0b76d77b
+          inside: "https://lob.com/snappackinside.pdf"
+          outside: "https://lob.com/snappackoutside.pdf"
+          size: 8.5x11
+          metadata:
+            spiffy: "true"
+          mail_type: usps_standard
+          merge_variables:
+            name: Harry
+          send_date: "2017-11-01T00:00:00.000Z"
+          use_type: marketing
+
+      multipart/form-data:
+        schema:
+          $ref: "models/snap_pack_editable.yml"
+        example:
+          description: Demo Snap Pack job
+          to: adr_bae820679f3f536b
+          from: adr_210a8d4b0b76d77b
+          inside: "https://lob.com/snappackinside.pdf"
+          outside: "https://lob.com/snappackoutside.pdf"
+          size: 8.5x11
+          metadata:
+            spiffy: "true"
+          mail_type: usps_standard
+          merge_variables:
+            name: Harry
+          send_date: "2017-11-01T00:00:00.000Z"
+          use_type: marketing
+
+  responses:
+    "200":
+      $ref: responses/post_snap_pack.yml
+
+    default:
+      $ref: "../../shared/responses/mailpiece_error.yml"
+
+  x-codeSamples:
```

Comment:
@richseviora tagging you to confirm whether the code samples align with the work you're doing on the SDKs.